.github/workflows/frontend.yml

```diff
@@ -3,6 +3,8 @@ name: Frontend CI
 on:
   pull_request:
     types: [opened, reopened, edited, synchronize]
+  push:
+    branches: master
```

Comment:
Can we make it a separate workflow that only triggers on changes to `packages/core`? :>